core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -57,10 +57,18 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
               story_id: ctrl.storyId
             });
           $http.get(reviewTestsDataUrl).then(function(result) {
+            var skillList = [];
+            var skillDescriptions = [];
+            for (var skillId in result.data.skills_with_description) {
+              skillList.push(skillId);
```

Comment:
This is a list of skill_ids right? If so, can we indicate that in the name?

Reply:
Done